packages/elements/src/tree/__test__/tree.test.js

```diff
@@ -490,6 +490,50 @@ describe('tree/Tree', () => {
       descendants.forEach(item => expect(el.manager.isItemHidden(item)).to.equal(false, 'Descendants of matched items must be included'));
     });
 
+    it('Text filter applied, single selection', async () => {
```

Comment:
I would update the test case name into something like 
```suggestion
    it('Should be able to select value after filter is applied', async () => {
```